### ISSUE TITLE
[SCAM ALERT] Backdoor in the code - Full control over user's PC / Multiple vulnerabilities

### ISSUE THREAD

**Issue by peterschorr** [terafoundation/wallet#5]
TL;DR **The code contains multiple backdoors to run arbitrary code on user's PC (with full access to files, processes, devices, etc.)**
____
There is a code to **trigger remote code execution vulnerability**. 
Developer is able to run specially crafted packed that will run **any** code on the target system.
Ref https://github.com/terafoundation/wallet/blob/master/Source/core/code.js#L239
____
Second vulnerability introduces:
* **remote code execution vulnerability**. 
* **remote write access to arbitrary files on the user's PC**

It downloads zip file over the network, unpacks it and restarts the process. As well as it might be used to edit/replace any file on the target system, it allows to replace the source code that will be run right after the process restart. Mean, using this vulnerability it is possible to replace source code with **arbitrary code and execute it** right after.
Ref https://github.com/terafoundation/wallet/blob/master/Source/core/code.js#L254

**Comment by terafoundation** [comment]
In the future, after the end of the development period, we will add a voting system for the adoption of a new version of the code.
